sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp

```diff
@@ -993,7 +1058,9 @@ namespace Azure { namespace Storage { namespace Blobs {
     /**
      * @brief Optional conditions that the source must meet to perform this operation.
      */
-    BlobAccessConditions SourceConditions;
+    struct : public LastModifiedTimeAccessConditions, public ETagAccessConditions
```

Comment:
I don't know if the unnamed struct pattern is a good idea for something that might be reused. Can we bring this to review before we decide that we can write code in this style?

Reply:
We defined separate structs for those that might be reused for example `BlobAccessConditions `. Access conditions for this API is very special. None of any other API has the same set of fields as it. So in this case I think it's fine to use unnamed struct. If in the future, we had a new API that share the same fields, we could define a new struct, it's not even a breaking change.